.travis.yml

```diff
@@ -2,5 +2,4 @@ sudo: false
 language: node_js
 
 node_js:
-  - "4"
-  - "6"
+  - "8"
```

Comment:
I would add "node" and/or "9" to be future proof

Reply:
Having both is the unofficial guidelines all hapi projects should follow.